test/functional/tool_wallet.py

```diff
@@ -87,15 +161,32 @@ def run_test(self):
             Address Book: 0
         ''')
         self.assert_tool_output(out, '-wallet=foo', 'create')
-
+        shasum_after = self.wallet_shasum()
```

Comment:
I understand surrounding the `info` methods, but what is the logic here? creating a wallet that already exists should leave it untouched?

Reply:
Yes, updated test name and logging to clarify that a bit. Happy to remove if you reckon it isn't a useful safeguard while fixing the issue.